### DISCUSSION TITLE
Mobile UI on Laptop for custom domain

### DISCUSSION THREAD

**Discussion by canaryGrapher** [vercel/community#630] [Help]
I hosted a website on Vercel that was created using nextJS+TypeScript+TailWind. The website has no errors or warnings, but the problem is with the custom domain. Through Vercel's assigned domain, the website is fine, but when I access it through Custom Domain, it shows me the Mobile UI of the page. It thinks that my PC is a Mobile Device. However, when I use the browser dev tools to access the website and view it as a large-screen device, the correct UI is rendered. However, when the website is run on the `localhost`, there is no such error. I deleted all browser-related cookies/sessions/storage but nothing seems to work. This issue only happens on my device.

**Comment by swarnava** [comment]
that shouldn't have happened. do you have a public repository which I can use to test this?

**Reply by canaryGrapher** [reply]
Here is the repository: https://github.com/canaryGrapher/portfolio-updated
